### ISSUE TITLE
Parallel CMFE Crash

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#1322]
Discovered when working with Ryan Vignes.
He has two multi-domain database that he us trying to use with a Pos-CMFE between to separate databases. 

Things work fine in serial, but in parallel the CMFE crashes.

Cyrus has the data on the CZ, and is investigating. 

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Cyrus Harrison
Original creation: 05/03/2013 05:39 pm
Original update: 05/03/2013 07:20 pm
Ticket number: 1447
